src/components/ResourceList/ResourceList.jsx

```diff
@@ -30,7 +31,7 @@ const InlineDiv = styled.div`
    {
     font-weight: normal;
     padding-left: 0px !important;
-    padding-top: 5px !important;
+    padding-top: ${spacing02} !important;
```

Comment:
Same here. This token does not map to 5px. What changes to the UI will this cause

Reply:
This will not break the UI, only brings the description slightly closer to the header when resource list design is set to inline so it can be standardized 👍